### PR TITLE
Refactor/style sidebar

### PR DESCRIPTION
**Pull request by jenovs**
Added a little styling to the Sidebar in the CMS view.

### CODE REVIEW

File: frontend/src/Sidebar/Sidebar.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ import MenuIcon from '@material-ui/icons/Menu';
 import expandTree from './expandTree';
 import buildHtml, { getTree, checkMobile } from './utils';
 
-import { Wrapper } from './styled';
+import { Dummy, Wrapper } from './styled';
```
Comment (nikrb) [inline frontend/src/Sidebar/Sidebar.js:12]: I just felt I should comment on `Dummy` - it just feels wrong :grinning:

**Review by nikrb — APPROVED** [review]
Not sure I like the vertical white gap between the sidebar items. Just wanted to point it out in case it wasn't intended.

**Comment by jenovs** [comment]
Feel free to change styling to your preferences. The default styling was such an eyesore so I just *had* to do something.
Regarding `Dummy`, it looks like some kind of a hack, but I was too lazy to try to figure out why it's there :grinning:

**Comment by nikrb** [comment]
see if anyone else comments on the styling, it maybe better and I'm just used to the old.
I added the dummy div I think, to pad the text from hitting the edges. Adjusting the width doesn't seem to have changed much.

**Pull request merged**